src/components/barchart/BarChart.tsx

```diff
@@ -48,6 +49,7 @@ const BarChart = ({
     exportingModule(Highcharts)
     offlineExporting(Highcharts)
   }
+  accessibilityModule(Highcharts)
```

Comment:
I happened to see a console warning saying we weren't using this so I decided to import this here to remove the warning, but I'm not entirely sure what it actually gives us